WordPress/Classes/ViewRelated/System/Search/WPSearchControllerConfigurator.m

```diff
@@ -0,0 +1,112 @@
+#import "WPSearchControllerConfigurator.h"
+#import "WordPress-Swift.h"
+
+const CGFloat SearchBarWidth = 280.0;
+const CGFloat SearchBariPadWidth = 600.0;
+const CGFloat SearchWrapperViewPortraitHeight = 64.0;
+const CGFloat SearchWrapperViewLandscapeHeight = 44.0;
+const NSTimeInterval SearchBarAnimationDuration = 0.2; // seconds
+
+@interface WPSearchControllerConfigurator ()
+
+@property (nonatomic, weak) WPSearchController *searchController;
+@property (nonatomic, weak) UIView *searchWrapperView;
+@property (nonatomic, weak) UISearchBar *searchBar;
+
+@end
+
+@implementation WPSearchControllerConfigurator
+
+- (instancetype)initWithSearchController:(WPSearchController *)searchController
+                   withSearchWrapperView:(UIView *)searchWrapperView
+{
+    self = [super init];
+    if (self) {
+        _searchController = searchController;
+        _searchWrapperView = searchWrapperView;
+        _searchBar = _searchController.searchBar;
+    }
+    
+    return self;
+}
+
+- (void)configureSearchControllerBarAndWrapperViewOfClass:(Class)class
```

Comment:
You were right, it was definitely unclear. Now, with the removal of the VC being passed in and the search bar being a property of the controller, I have renamed it to `configureSearchControllerAndWrapperView`. I believe this is clear because it is initialized with a `searchController` and `wrapperView`.